### ISSUE TITLE
Visuality reforged not working

### ISSUE THREAD

**Issue by pompersuf** [DragonsPlusMinecraft/VisualityReforged#23]
i have visuality reforged  added to the mod list and none of the particle effects are working but they were working before so are there any mods that are not compatible or any resource packs

**Comment by Wolfcvz** [comment]
Same problem in 1.19.2

**Comment by DrethPlusPlus** [comment]
+1 on 1.20.1, also suspecting resource packs, specifically those using emf and etf (fresh animations)

**Comment by Jusey1** [comment]
I don't think it is EMF or ETF cuz I am having the issue without anything like that.

**Comment by Jusey1** [comment]
Just tested this out in Neoforge 1.21.1:

1. Particle effects on armor, such as golden & diamond, don't work outright. Not sure what is going on there.
2. Everything else works as expected until the Explosive Enhancement Reforged mod is installed, which can be found at: https://legacy.curseforge.com/minecraft/mc-mods/explosive-enhancement-reforged

**Comment by Jusey1** [comment]
This just got fixed today by Explosive Enhancement Reforged, I am now seeing Visuality effects again, except for armor!

**Comment by argo20k** [comment]
I seem to be experiencing the same issue in regards to
- armour not having it's sparkle effect

would love to see this working 👍
thanks for working on this great mod

**Comment by VaporeonScripts** [comment]
+1 , the mod is not working at all
